material-flux/js/actions/Actions.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+import { Action } from 'material-flux';
+import { checkoutProducts, getAllProducts } from '../utils/WebAPIUtils';
+export var keys = {
+    "getAllProducts": "getAllProducts",
```

Comment:
@azu please change to single quotes, see [CONTRIBUTING.md](https://github.com/voronianski/flux-comparison/blob/master/CONTRIBUTING.md).